avionics/common/src/radio.cpp

```diff
@@ -88,15 +88,28 @@ class Radio::RadioMembers {
     ZBTxRequest tx_packet_;
     ZBRxResponse rx;
 
+    std::unordered_map<StateId, EventId> state_change_events;
+
     RadioQueue tx_q_; // the [ueue] is silent :)
     uint8_t payload_[RADIO_MAX_SUBPACKET_SIZE];
 };
 static Radio::RadioMembers self;
 
 constexpr Hal::ms Radio::WATCHDOG_SEND_INTERVAL;
 
+// Initializes radio member variables;
 Radio::RadioMembers::RadioMembers()
-    : gnd_addr_(GND_STN_ADDR_MSB, GND_STN_ADDR_LSB), tx_q_(MAX_QUEUED_BYTES) {}
+    : gnd_addr_(GND_STN_ADDR_MSB, GND_STN_ADDR_LSB),
+      tx_q_(MAX_QUEUED_BYTES),
+      state_change_events ({
```

Comment:
Feels kind of awkward that this mapping is with Radio, but more importantly radio is common code but what states are available is rocket specific, so maybe this mapping isn't always correct for different rockets.  Ideally this mapping should be done in the rocket-specific state machine, but I had some ideas for refactoring the state machine so that a common central object can do specific actions on entry and exit of states. 

One idea - change the spec of events so e.g. the first byte is a classifier, e.g. state change report, and have the next byte indicate more detail.

In the meantime - it might be better to instead hold off for now for reporting state events - probably more important is reporting ignitor events